simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/PerfTestLarge.scala

```diff
@@ -54,8 +60,8 @@ class PerfTestLarge extends SimulationWithScenario {
-			.formParam("transaction_uuid", "feaf75a3-c82c-4e06-9d02-5387f14034a0")
+			.formParam("facts", "%7B%22name%22%3A%22${node}%22%2C%22values%22%3A%7B" + "${Fact}" + "${HostFact}" + facts_str)
```

Comment:
No, I mentioned it in Doug's PR, but we decided to just keep it encoded since it was a static, one time thing. If we were constantly updating the values, then having it auto url-encode would be the way to go.